src/components/views/elements/AccessibleButton.tsx

```diff
@@ -39,10 +39,11 @@ interface IProps extends React.InputHTMLAttributes<Element> {
     tabIndex?: number;
     disabled?: boolean;
     className?: string;
-    onClick(e?: ButtonEvent): void;
+    // Optional in case we need to use onMouseDown
+    onClick?(e?: ButtonEvent): void;
```

Comment:
this change is a bit worrisome: the point of using `onClick` as a property is to ensure that the component can control it appropriately. If another component were to bypass this, then the component is considered to have lost control of its click handling.

A new prop for `triggerOnMouseDown = true` is ugly but maintains the component's desired capability.